src/main/res/values/strings.xml

```diff
@@ -586,185 +587,185 @@
     <!-- User Interface Settings -->
-    <string name="config_category_view">User Interface Settings</string>
-    <string name="config_category_view_summary">Layout and display configuration options</string>
+    <string name="config_category_view">User interface settings</string>
+    <string name="config_category_view_summary">Layout and display configuration options.</string>
     <string name="config_iconbutton_title">Show node icons</string>
     <string name="config_iconbutton_summary">Show the icon of the best-matching preset above nodes.</string>
     <string name="config_showWayIcons_title">Show POI icons on buildings</string>
     <string name="config_showWayIcons_summary">Show the icon of best-matching preset above buildings.</string>
     <string name="config_tagFormEnabled_title">Display tag form</string>
-    <string name="config_tagFormEnabled_summary">Display tag form for editing</string>
+    <string name="config_tagFormEnabled_summary">Display tag form for editing.</string>
     <string name="config_showCameraAction_title">Show camera action</string>
-    <string name="config_showCameraAction_summary">Show camera icon on action bar</string>
-    <string name="config_followGPSbutton_title">Follow GPS button layout</string>
-    <string name="config_followGPSbutton_summary">Where the "follow GPS" button will be positioned</string>
+    <string name="config_showCameraAction_summary">Show camera icon on action bar.</string>
+    <string name="config_followGPSbutton_title">Follow positional device button layout</string>
+    <string name="config_followGPSbutton_summary">Where the "follow positional device" button will be positioned.</string>
     <string name="config_alwaysDrawBoundingBoxes_title">Always dim non-downloaded areas</string>
-    <string name="config_alwaysDrawBoundingBoxes_summary">If set non-downloaded areas will be dimmed in locked mode</string>
-    <string name="config_fullscreenMode_title">Full screen mode</string>
+    <string name="config_alwaysDrawBoundingBoxes_summary">Dims non-downloaded areas in locked mode.</string>
+    <string name="config_fullscreenMode_title">Fullscreen mode</string>
     <string name="config_fullscreenMode_summary">Behaviour on devices with "soft" keys.</string>   
     <string name="config_showTolerance_title">Show tolerance</string>
-    <string name="config_showTolerance_summary">Displays the area where an element can be selected.</string>
-    <string name="config_use_back_for_undo_title">Use back key for Undo</string>
-    <string name="config_use_back_for_undo_summary">Use the back key for undo in the main edit view</string>
-    <string name="config_enableLightTheme_title">Enable light theme</string>
-    <string name="config_enableLightTheme_summary">Light theme with white backgrounds</string>
+    <string name="config_showTolerance_summary">Displays area where an element can be selected.</string>
+    <string name="config_use_back_for_undo_title">Use ← key to undo</string>
```

Comment:
Some strings reference "key" and some "button", I imagine key relates to keyboard and the latter to its respective in-app button…? Not sure which is which from this context.

Reply:
We've typically used "key" to refer to hardware keys, but I don't believe that has been very consistent.